test/__tests__/data/criteo_audience_router_input.json

```diff
@@ -0,0 +1,111 @@
+[
```

Comment:
Are we planning to add more test-cases to cover couple of failure scenarios in transformation ?

Reply:
We have covered the possible test cases. Are you talking about the errors the transformer will be throwing?

Reply:
Yes during transformation, we'll have some errors thrown right ?!
And also errneous responses that we might get from destination

Reply:
Can we add cases to cover errors thrown during transformation ?

Reply:
Okay

Reply:
Done